### DISCUSSION TITLE
Text is no longer highlighted when actions menu opens

### DISCUSSION THREAD

**Discussion by rhoadsstevens** [ManifoldScholar/manifold#3480] [Accessibility]
Priority level 1 (out of 3) from UW Accessible Technologies

WCAG mapping: 1.3.1 Info Relationships

STEPS TO REPRODUCE:
1. Select a book (i.e. "I My Recall")
2. Turn on caret browsing (insert + F7)
3. Tab into the text
4. Arrow through the text, hold shift as using arrow keys to highlight something.
5. Observe the text when the context menu  appears.

OBSERVED BEHAVIOR
When highlighting the text to create a highlight/annotation, once you release the modifier keys (shift or shift/ctrl) and the context menu appears, the text that has been selected is no longer highlighted.

EXPECTED BEHAVIOR:
The text should remain highlighted when the context menu opens so that users are able to confirm the text that they have selected.

**Comment by dananjohnson** [comment]
This was resolved in v8.
